framework/TestFiles.py

```diff
@@ -0,0 +1,244 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+  This Module performs Unit Tests for the Files class.
+  It cannot be considered part of the active code but of the regression test system
+"""
+
+#For future compatibility with Python 3
+from __future__ import division, print_function, unicode_literals, absolute_import
+import warnings
+warnings.simplefilter('default',DeprecationWarning)
+
+import sys, os
+import cPickle as pk
+import numpy as np
+import xml.etree.ElementTree as ET
+frameworkDir = os.path.dirname(os.path.abspath(os.path.join(sys.argv[0])))
+
```

Comment:
remove 'empty line' if unnecessary for this whole file

Reply:
I categorically disagree; I feel empty space is critical to parsing the code quickly for users and developers.

Reply:
I agree that empty line can be useful. What I tried to say is that double check this file, and make sure all empty lines are necessary, otherwise remove them.